stonesoup/platform/simple.py

```diff
@@ -43,90 +55,111 @@ def __init__(self, *args, **kwargs):
         consistent at initialisation.
         """
         super().__init__(*args, **kwargs)
-        if self.mounting_mappings.max() > len(self.state.state_vector):
+        # Set values to defaults if not provided
+        if self.mounting_offsets is None:
+            self.mounting_offsets = [StateVector([0] * self.ndim)] * len(self.sensors)
+
+        if self.rotation_offsets is None:
+            self.rotation_offsets = [StateVector([0] * 3)] * len(self.sensors)
+
+        if self.mounting_mappings is None:
+            self.mounting_mappings = self.mapping
+
+        if ((isinstance(self.mounting_mappings, Sequence) and self.mounting_mappings)
+            and (isinstance(self.mounting_mappings[0], np.ndarray) or
+                 isinstance(self.mounting_mappings[0], Sequence))):
+            # We were passed a non-empty list of arrays or lists
+            self.mounting_mappings = [coerce_to_valid_mapping(m) for m in self.mounting_mappings]
+        elif (isinstance(self.mounting_mappings, np.ndarray) or
+              isinstance(self.mounting_mappings, Sequence)):
+            # We were passed either list of non-arrays (assumed to be ints) or a single array, so
+            # coerce the single entry and then expand
+            # noinspection PyTypeChecker
+            single_mapping = coerce_to_valid_mapping(self.mounting_mappings)
+            self.mounting_mappings = [single_mapping] * len(self.sensors)
```

Comment:
I still think it would be better to just required mountings to be list of mountings. How often would you actually have multiple sensors mounted at the exact same point?

Reply:
Might you not have many sensors with negligibly different mountings, and hence want the all at (say) the nose of the platform? I agree that it complicates the code in this class though, so if it's a minor benefit, maybe better to enforce the List[mapping] approach.

Reply:
Actually: I think there is some confusion here (which is not a good sign!). `mounting_mapping` is a list of mappings between state vectors and positions (analogous to `mapping`/`position_mapping` in the platform). The single entry option was added because these would **almost always** be the same. 

I will review whether these are now necessary given the new architecture, and even if they are needed, perhaps the default that they are the same as the platform mapping is good enough, meaning the single StateVector option can be dropped.

`mounting_offsets` specify where on the platform the sensor is mounted, and must be a list (defaulting to a list of zero vectors).

Reply:
I have removed mounting mappings in a new version.